pkgs/development/ocaml-modules/pp/default.nix

```diff
@@ -0,0 +1,28 @@
+{ buildDunePackage
+, fetchzip
+, ppx_expect
+, lib
+}:
+
+buildDunePackage rec {
+  pname = "pp";
+  version = "1.1.2";
+
+  src = fetchzip {
+    url = "https://github.com/ocaml-dune/pp/releases/download/${version}/pp-${version}.tbz";
```

Comment:
Why don’t you use `fetchurl`?

Reply:
I use those quite interchangeably, can you tell me where/when I should use one and not the other ?